worlds/wrestling.wbt

```diff
@@ -1,4 +1,4 @@
-#VRML_SIM R2023a utf8
+#VRML_SIM R2023b utf8
```

Comment:
Do we want to set the version to R2023b or should we set it to the last stable version?

Reply:
Well spotted!